common/djangoapps/entitlements/models.py

```diff
@@ -257,3 +257,28 @@ def set_enrollment(self, enrollment):
     @classmethod
     def unexpired_entitlements_for_user(cls, user):
         return cls.objects.filter(user=user, expired_at=None).select_related('user')
+
+    @classmethod
+    def get_entitlement_if_active(cls, user, course_uuid):
+        """
+        Returns an entitlement for a given course uuid if an active entitlement exists, otherwise returns None.
+        An active entitlement is defined as an entitlement that has not yet expired or has a currently enrolled session.
+        """
+        return cls.objects.filter(user=user, course_uuid=course_uuid)\
+            .exclude(expired_at__isnull=False, enrollment_course_run=None).first()
```

Comment:
Avoid using the \ in this case it is preferable to do the following.

```
return cls.objects.filter(
    user=user,
    course_uuid=course_uuid
).exclude(expired_at__isnull=False, enrollment_course_run=None).first()
```

Reference: http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#breaking-long-lines